pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -202,19 +202,7 @@ func (w *Webhook) inject(pod *corev1.Pod, ns string, _ dynamic.Interface) (bool,
 		}
 	}
 
-	// Inject env variables used for Onboarding KPIs propagation
-	var injectionType string
```

Comment:
This was being done twice, once in `injectAutoInstruConfig` -- already covered by existing tests for `v1` and added more tests for this in `v2`.